src/edu/wright/cs/raiderplanner/view/Activity.fxml

```diff
@@ -2,13 +2,13 @@
 
 <?import javafx.scene.image.*?>
 <?import java.lang.*?>
-<?import javafx.scene.text.*?> 
+<?import javafx.scene.text.*?>
```

Comment:
Only change what you're doing. You removed <?import javafx.scene.text.*?> and replaced it with <?import javafx.scene.text.*?>. Below you did the same. Why?